parser/prism/Translator.cc

```diff
@@ -650,6 +650,23 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
 
             return make_unique<parser::Kwrestarg>(location, sorbetName);
         }
+        case PM_LAMBDA_NODE: { // lambda literals, like `-> { 123 }` or `lambda { 123 }`
+            auto lambdaNode = reinterpret_cast<pm_lambda_node *>(node);
+
+            auto params = translate(reinterpret_cast<pm_node *>(lambdaNode->parameters));
+            auto body = translateStatements(reinterpret_cast<pm_statements_node *>(lambdaNode->body), true);
+            auto receiver = make_unique<parser::Const>(location, nullptr, core::Names::Constants::Kernel());
+
+            auto sendNode = make_unique<parser::Send>(location, move(receiver), core::Names::lambda(),
+                                                      translateLoc(lambdaNode->operator_loc), NodeVec{});
+
+            if (lambdaNode->parameters != nullptr &&
```

Comment:
This is really similar to `translateCallWithBlock`. Is it possible to tweak that hepler (or extract a new one?) to support both use cases?